mongodb-panache-quickstart/pom.xml

```diff
@@ -6,10 +6,11 @@
   <artifactId>mongodb-panache-quickstart</artifactId>
   <version>1.0-SNAPSHOT</version>
   <properties>
-    <surefire-plugin.version>2.22.0</surefire-plugin.version>
+    <quarkus.version>999-SNAPSHOT</quarkus.version>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <surefire-plugin.version>2.22.1</surefire-plugin.version>
```

Comment:
Is it really needed to fix these versions ? Are the default ones comming from the parent OK ?